### PR TITLE
Refactor/renaming json files to new naming convention 🔄️

### PR DESCRIPTION
**Pull request by pbullhove**
## What does this pull request change?

In the /example/ app: 
- Blueprints are now <BluePrintName>.blueprint.json
- Recipes are now <BluePrintName>.recipe.json
- Entities are now <entityName>.entity.json

## Why is this pull request needed?

In order to more easiliy differentiate between json files. 

## Issues related to this change

[Issue #270](https://github.com/equinor/dm-core-packages/issues/270)

### CODE REVIEW

File: example/app/data/DemoDataSource/recipes/apps/EmployeeApp/Employee.recipe.json
Comment (ingeridhellen) [inline example/app/data/DemoDataSource/recipes/apps/EmployeeApp/Employee.recipe.json:1]: recipes should have camelcase, just as entities

**Review by ingeridhellen — COMMENTED** [review]
The files in dm-core-packages/example/app/data/DemoDataSource/apps/EmployeeApp have not been renamed

File: example/app/data/DemoDataSource/recipes/apps/EmployeeApp/Employee.recipe.json
Comment (pbullhove) [reply example/app/data/DemoDataSource/recipes/apps/EmployeeApp/Employee.recipe.json:1]: fixed

**Review by ingeridhellen — APPROVED** [review]
EmployeeApp

**Pull request merged**